src/Command/BinCommand.php

```diff
@@ -94,7 +94,8 @@ public function isProxyCommand(): bool
 
     public function execute(InputInterface $input, OutputInterface $output): int
     {
-        $config = Config::fromComposer($this->requireComposer());
+        // Switch to requireComposer() once Composer 2.3 is set as the minimum
+        $config = Config::fromComposer($this->getComposer());
```

Comment:
@theofidry maybe add a check if `requireComposer` exist and if so, use it?

that way it would continue working in potential newer composer version which might not have `getComposer` anymore. 

Though as it is a breaking change it probably won't happen prior to composer 3.x and until then there will most likely be a plugin version for 2.3 or higher,, so it is probably unnecessary.

Reply:
Done in #135